containers/Dockerfile.buildenv

```diff
@@ -74,6 +75,7 @@ RUN add-apt-repository universe \
     && apt-get install -y curl python2 python2-dev \
     && curl https://bootstrap.pypa.io/get-pip.py --output get-pip.py \
     && python2 get-pip.py \
+    && apt-get update -y \
```

Comment:
this one is not necessary because there's an `apt-get update` in this same `RUN` layer on line 74

Reply:
Oh, I see, I misunderstood your earlier comment - thanks for clarifying. I'll remove this one :)

Reply:
Here's the docs on that BTW if you're interested https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run